decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/util/HTTPSCustomTruststore.java

```diff
@@ -33,7 +33,7 @@
      */
     public static void configureTruststore(HttpsURLConnection connection, String truststorePath, String pwd)
             throws Exception {
-        KeyStore keystore = KeyStore.getInstance("JKS");
+        KeyStore keystore = KeyStore.getInstance("PKCS12");
```

Comment:
Maybe the type of Keystore should be set as a parameter or configured somehow?

Reply:
I changed it to the default keystore type